.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -43,8 +37,8 @@ body:
     attributes:
       label: "Steps to reproduce"
       description: |
-        Steps to reproduce using `nvim -u NORC` and/or `nvim -u NONE` (please test both).
-        If you are reporting build failures, please list the exact sequence of steps including all CMake flags (if any).
+        If you cannot reproduce the issue using `nvim -u NORC`, prepare and upload a [minimal config](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua) that reproduces the issue with `nvim -u minimal.lua`.
```

Comment:
It might be obvious to us, but I think we could benefit from expanding on this a little. Reminding the issuer that the smaller the configuration is, the more likely we'll be able to pinpoint the problem and the more likely we'll be able to resolve their issue ("help us help you"). I know this is already implied by the "minimal" part but my hope is that expressing that we _want_ to solve their issues but need their help might encourage better reports.

Reply:
That _would_ be useful to explain, but probably not here. We could add a section to `CONTRIBUTING.md` and link that?

Reply:
Hmm yeah, probably a bit too wordy. I don't think we need to link it honestly, it'll just add noise. I'm backpedaling on this one, think we can drop it.

Reply:
What? I need to have permission to resolve _my own_ reviews?

Reply:
No, if we _do_ add such a section, it could replace the link to the file. It's something I'll think about.

Reply:
(Probably just a conflict because we commented at the same time.)